src/components/Popup.js

```diff
@@ -57,7 +58,25 @@ export default class Popup {
 
   setEventListeners() {
     const closeButton = this._popupElement.querySelector(".modal__close");
-
     closeButton.addEventListener("click", () => this.close());
   }
+
+  //
```

Comment:
`setButtonText` and `resetButtonText` should be removed from `Popup`

There should **not** be any code in universal functions/classes that doesn’t suit every popup.